src/components/add-transaction-dialog/add-transaction-dialog.html

```diff
@@ -0,0 +1,116 @@
+<aside class="mdc-dialog add-transaction-dialog" role="alertdialog" aria-labelledby="my-mdc-dialog-label" aria-describedby="my-mdc-dialog-description">
+  <div class="mdc-dialog__surface">
+
+    <header class="mdc-dialog__header">
+      <h2 class="mdc-dialog__header__title">
+        Add transaction
+      </h2>
+    </header>
+
+    <section class="mdc-dialog__body">
+      <div class="mdc-layout-grid add-transaction-dialog__row">
+        <div class="mdc-layout-grid__inner">
```

Comment:
you do not need grid in this dialog -- it adds unneeded padding inside dialog

Reply:
this is the Material Components way to position elements. How should this dialog look on your suggestion? all in one column?

Reply:
it adds unneeded padding inside dialog. Try to use flex/inline-block/floats instead of grid